packages/design-picker/src/__tests__/integration.test.tsx

```diff
@@ -1,17 +1,9 @@
-/**
- * External dependencies
- */
-import * as React from 'react';
 import { render, screen, fireEvent } from '@testing-library/react';
-import '@automattic/calypso-config';
```

Comment:
I'm curious if this needs to stay -- it's mocked with `jest.mock` below. I don't know much about how that works, but I wonder if it's imported intentionally.

Reply:
Tests are passing without this line, so I figured it wasn't important. As far as I know, you can mock any module that is eventually required without having to require it directly in the test code.